drivers/serial/uart_stm32.c

```diff
@@ -589,11 +589,17 @@ static int uart_stm32_err_check(const struct device *dev)
 	if (LL_USART_IsActiveFlag_FE(UartInstance)) {
 		err |= UART_ERROR_FRAMING;
 	}
-
+	
+	#if !defined(CONFIG_SOC_SERIES_STM32F0X) || defined(USART_LIN_SUPPORT)
```

Comment:
Thanks for the update.
Please move "#ifdef'ery" to the beginning of the line.

Reply:
@andreacampanella Also, please do your change in only one commit as this is one single and coherent change.